### PR TITLE
[Minor] Assorted cleanup and housekeeping of configuration files, take 2

### PR DESCRIPTION
**Pull request by twesterhever**
This merge request improves and streamlines a variety of rule descriptions and comments. Similar to https://github.com/rspamd/rspamd/pull/4397, it does not introduce any functional changes.

### CODE REVIEW

File: rules/html.lua [hunk 1]
```diff
@@ -21,7 +21,7 @@ local rspamd_regexp = require "rspamd_regexp"
 reconf['MIME_HTML_ONLY'] = {
   re = 'has_only_html_part()',
   score = 0.2,
-  description = 'Messages that have only HTML part',
+  description = 'Messages has only an HTML part',
```
Comment (moisseev) [inline rules/html.lua:24]: Is `message` supposed to be singular?

Comment (vstakhov) [reply rules/html.lua]:
```suggestion
  description = 'The message has only an HTML part',
```

File: rules/html.lua [hunk 2]
```diff
@@ -395,7 +395,7 @@ rspamd_config.HTTP_TO_HTTPS = {
     end
     return false
   end,
-  description = 'Anchor text contains different scheme to target URL',
+  description = 'HTML anchor text contains different scheme to target URL',
```
Comment (vstakhov) [inline rules/html.lua:398]:
```suggestion
  description = 'The anchor text contains a distinct scheme compared to the target URL',
```

File: rules/misc.lua [hunk 3]
```diff
@@ -148,7 +148,7 @@ local obscured_id = rspamd_config:register_symbol{
   name = 'R_SUSPICIOUS_URL',
   score = 5.0,
   one_shot = true,
-  description = 'Obfuscated or suspicious URL has been found in a message',
+  description = 'Has obfuscated or suspicious URL',
```
Comment (vstakhov) [inline rules/misc.lua:151]:
```suggestion
  description = 'A message has been identified to contain an obfuscated or suspicious URL',
```

File: rules/html.lua (hunk 1 above)
Comment (twesterhever) [reply rules/html.lua]: Missed that, thank you. Pushed a fix to this branch.

File: rules/misc.lua [hunk 4]
```diff
@@ -382,7 +382,7 @@ end
 rspamd_config:register_symbol({
   name = 'FREEMAIL_REPLYTO_NEQ_FROM_DOM',
   callback = freemail_reply_neq_from,
-  description = 'Freemail From and Reply-To, but to different Freemail services',
+  description = 'Freemail addresses in From and Reply-To headers, but to different freemail services',
```
Comment (vstakhov) [inline rules/misc.lua:385]:
```suggestion
  description = 'The From and Reply-To addresses in the email are from different freemail services',
```

**Review by vstakhov — APPROVED** [review]
LGTM with some minor suggestions

**Pull request merged**